src/org/freenetproject/freemail/wot/IdentityMatcher.java

```diff
@@ -38,23 +38,23 @@ public IdentityMatcher(WoTConnection wotConnection) {
 		this.wotConnection = wotConnection;
 	}
 
-	public Map<String, List<Identity>> matchIdentities(Set<String> recipients, String wotOwnIdentity, EnumSet<MatchMethod> methods) throws PluginNotFoundException {
-		Set<Identity> trustedIds = wotConnection.getAllTrustedIdentities(wotOwnIdentity);
-		Set<Identity> untrustedIds = wotConnection.getAllUntrustedIdentities(wotOwnIdentity);
+	public Map<String, List<Identity>> matchIdentities(
+		 Set<String> recipients, String wotOwnIdentity, EnumSet<MatchMethod> methods)
```

Comment:
Indentation with a tab and a space? That doesn’t look correct (or good). :)

Reply:
This is a long line that I split to make it easier to read.